web/src/lib/dragdrop/provider.tsx

```
@@ -111,20 +112,48 @@ export function DndProvider({ children }: { children: React.ReactNode }) {
+                  ? target.direction
+                  : ("inside" as const),
+              relativeToNode:
+                target.type === "divider"
+                  ? target.siblingNode.id
+                  : target.node.id,
+              newParentNode:
+                target.type === "divider" ? target.parentID : target.node.id,
+            };
+
+          case "node-children":
+            const isTop = target.sortable.index === 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)